topics/introduction/slides/index.html

```diff
@@ -35,10 +35,18 @@
   - Users can share or publish their analyses (histories, workflows, visualizations)
   - Pages: online Methods for your paper
 
+Pages: interactive, web-based documents that describe a complete analysis.
+
 ???
 
-- Information captured by Galaxy include: tool version, inputs and parameters.
-- Pages: interactive, web-based documents that describe a complete analysis.
+**accessible** **reproducible** **transparent** research means <i>sharing everything</i>.
+
+If the Galaxy framework makes everything as simple as possible, researchers are able to:
+- share their analyses
+- track all used tools and versions
+- check all parameters
+- justify each step in the analysis
+- publish the findings with all aforementioned informations
```

Comment:
s/informations/information/ , it has no plural in English.

Reply:
thanks, @nsoranzo!